cmd/smith/app/app.go

```diff
@@ -207,10 +210,35 @@ func NewFromFlags(flagset *flag.FlagSet, arguments []string) (*App, error) {
 	flagset.IntVar(&a.Workers, "workers", 2, "Number of workers that handle events from informers")
 	flagset.StringVar(&a.Namespace, "namespace", meta_v1.NamespaceAll, "Namespace to use. All namespaces are used if empty string or omitted")
 	pprofAddr := flag.String("pprof-address", "", "Address for pprof to listen on")
+	printBundleSchema := flag.String("print-bundle-schema", "", "Print Bundle schema and exit (specify format: json or yaml)")
```

Comment:
what do you need this application mode for? debugging?

Reply:
To get the schema in json or yaml? Because I don't want to write it by hand.

Reply:
should be a separate binary then? :)
just a bit confusing given that it's not part of "normal" usage of the application.

Reply:
Yeah, I thought about making it a separate binary. Let me try to see how it'll look...

Reply:
Looks better now!